packages/js/package.json

```diff
@@ -33,7 +37,7 @@
   "private": true,
   "scripts": {
     "start": "pnpm run build -- --watch --sourcemap",
-    "build": "tsup && pnpm run build:umd && pnpm run post:build",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** can we revert this change?‏

Reply:
I'm letting tsup generate UMD now. We can revisit this though. Didn't touch the webpack config at all.